images-list

```diff
@@ -535,12 +539,14 @@ quay.io/k8scsi/csi-attacher rancher/mirrored-k8scsi-csi-attacher v3.0.0
 quay.io/k8scsi/csi-attacher rancher/mirrored-k8scsi-csi-attacher v3.1.0
 quay.io/k8scsi/csi-node-driver-registrar rancher/mirrored-k8scsi-csi-node-driver-registrar v2.0.1
 quay.io/k8scsi/csi-node-driver-registrar rancher/mirrored-k8scsi-csi-node-driver-registrar v2.1.0
+quay.io/k8scsi/csi-node-driver-registrar rancher/mirrored-k8scsi-csi-node-driver-registrar v2.3.0
 quay.io/k8scsi/csi-provisioner rancher/mirrored-k8scsi-csi-provisioner v2.0.0
 quay.io/k8scsi/csi-provisioner rancher/mirrored-k8scsi-csi-provisioner v2.1.0
 quay.io/k8scsi/csi-resizer rancher/mirrored-k8scsi-csi-resizer v1.0.0
 quay.io/k8scsi/csi-resizer rancher/mirrored-k8scsi-csi-resizer v1.1.0
 quay.io/k8scsi/livenessprobe rancher/mirrored-k8scsi-livenessprobe v2.1.0
 quay.io/k8scsi/livenessprobe rancher/mirrored-k8scsi-livenessprobe v2.2.0
+quay.io/k8scsi/livenessprobe rancher/mirrored-k8scsi-livenessprobe v2.4.0
```

Comment:
This is incorrect, the chart is using a [new image](https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/28c55a22dcb986460b66f7de3838c96146e39900/manifests/vanilla/vsphere-csi-driver.yaml#L307) from a different registry so a new repo will need to be created and the image will need a new name. This should be:
`k8s.gcr.io/sig-storage/livenessprobe:v2.4.0 rancher/mirrored-sig-storage-livenessprobe v2.4.0`